nototools/noto_cmap_reqs.py

```diff
@@ -355,6 +355,17 @@ def _primary_script_for_block(block):
   return _block_to_primary_script[block]
 
 
+def _remove_unicode_assignments(cmap_ops):
+  """The starting point is based on the script and script extensions data from
+  Unicode.  Sometimes the assignments seem premature."""
+  cmap_ops.phase('remove unicode assignments')
+
+  # Jelle says A8F1 makes no sense for Bengali since other characters needed
+  # for cantillation are not defined.  Unicode script extensions assign it to
+  # Deva and Beng, leave it for Deva.
+  cmap_ops.remove(0xa8f1, 'Beng')
```

Comment:
There seems to be stuff going on in this space.  There's a proposal from Jan of this year to add "combining bengali sign avagraha" to Bengali with "the same properties as A8F1".  I asked Roozbeh if he knew more about it.  We'll see I guess.